docs/book/customers/map.rst.inc

```diff
@@ -1,8 +1,4 @@
-.. raw:: html
-
-    <ul class="simple">
-    <li><p><a class="reference internal" href="customers/customer_and_shopuser.html"><span class="doc">Customer and ShopUser</span></a></p></li>
-    <li><p><a class="reference internal plus-feature" href="customers/customer_pools.html"><span class="doc">Customer Pools</span></a></p></li>
-    <li><p><a class="reference internal" href="customers/adminuser.html"><span class="doc">AdminUser</span></a></p></li>
-    <li><p><a class="reference internal" href="customers/addresses/index.html"><span class="doc">Addresses</span></a></p></li>
-    </ul>
\ No newline at end of file
+* :doc:`/book/customers/customer_and_shopuser`
+* :doc:`/book/customers/customer_pools`
+* :doc:`/book/customers/adminuser`
```

Comment:
```suggestion
* :doc:`/book/customers/admin_user`
```

Reply:
sorry, it's the filename, I cannot change it in the scope of this PR.